datepicker-react/package.json

```diff
@@ -1,32 +1,44 @@
 {
-    "name": "datepicker-demo",
-    "version": "0.1.0",
-    "private": true,
-    "dependencies": {
-        "body-parser": "^1.18.3",
-        "concurrently": "^3.6.0",
-        "dotenv": "^6.0.0",
-        "express": "^4.16.3",
-        "http-proxy": "^1.17.0",
-        "moment": "^2.22.2",
-        "nodemon": "^1.17.5",
-        "qs": "^6.5.2",
-        "react": "^16.4.1",
-        "react-datepicker": "^1.5.0",
-        "react-dom": "^16.4.1",
-        "react-helmet": "^5.2.0",
-        "react-router-dom": "^4.3.1",
-        "react-scripts": "1.1.4",
-        "smooch": "^4.12.9",
-        "smooch-core": "8.4.0",
-        "styled-components": "^3.3.3"
-    },
-    "scripts": {
-        "start": "concurrently \"npm run start:app\" \"npm run start:server\"",
-        "start:app": "react-scripts start",
-        "start:server": "nodemon server/index.js",
-        "build": "react-scripts build",
-        "test": "react-scripts test --env=jsdom",
-        "eject": "react-scripts eject"
-    }
+  "name": "datepicker-demo",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "concurrently": "^3.6.0",
+    "dotenv": "^6.0.0",
+    "express": "^4.16.3",
+    "http-proxy": "^1.18.1",
+    "moment": "^2.22.2",
+    "nodemon": "^2.0.6",
+    "qs": "^6.5.2",
+    "react": "^17.0.1",
+    "react-datepicker": "^1.5.0",
+    "react-dom": "^17.0.1",
+    "react-helmet": "^5.2.0",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "4.0.1",
+    "smooch": "^4.12.9",
+    "styled-components": "^5.2.1",
+    "sunshine-conversations-client": "^9.4.1"
+  },
+  "scripts": {
+    "start": "concurrently \"npm run start:app\" \"npm run start:server\"",
+    "start:app": "react-scripts start",
+    "start:server": "nodemon server/index.js",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  }
```

Comment:
Was it added intentionally?

Reply:
Nope, it is a remnant of the past. Don't know if we need this at all

Reply:
Maybe a package post install hook added this? I think it's meant to be a standardized way to document which browsers are supported: https://github.com/browserslist/browserslist-example#how-to-add-browserslist-to-your-project . But I'd vote to omit it since it's only for demo purpose 🤷

Reply:
Removed from package.json